resources/wikia/modules/iframeWriter.js

```diff
@@ -28,6 +28,11 @@ define('wikia.iframeWriter', [
 		iframe.scrolling = 'no';
 		iframe.width = params.width;
 		iframe.height = params.height;
+
+		if (iframe.classList) {
+			iframe.classList.add('wikia-ad-iframe');
```

Comment:
+1 to adding an optional class param to iframeWriter

On 03/25/2015 11:57 AM, Andrzej wrote:
> In resources/wikia/modules/iframeWriter.js
> <https://github.com/Wikia/app/pull/6758#discussion_r27110626>:
>
>> @@ -28,6 +28,11 @@ define('wikia.iframeWriter', [
>>  		iframe.scrolling = 'no';
>>  		iframe.width = params.width;
>>  		iframe.height = params.height;
>> +
>> +		if (iframe.classList) {
>> +			iframe.classList.add('wikia-ad-iframe');
>
> After giving it a second thought to this I'm wondering if it shouldn't
> be |wikia-iframe| or if we shouldn't add possibility to pass classes in
> the |params| object.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/app/pull/6758/files#r27110626>.
>